fakestorage/upload_test.go

```diff
@@ -596,23 +592,6 @@ func TestServerClientSignedUploadBucketCNAME(t *testing.T) {
 	if resp.StatusCode != http.StatusOK {
 		t.Errorf("wrong status returned\nwant %d\ngot  %d", http.StatusOK, resp.StatusCode)
 	}
-	data, err := io.ReadAll(resp.Body)
```

Comment:
This was incorrect, the XML APIs should not return a body and certainly shouldn't return JSON

Reply:
🤔 Oh are signed uploads powered by the XML API? Interesting!